### PR TITLE
Portfolio Feature: Desktop version

### PR DESCRIPTION
**Pull request by AkshithaReddy1899**
# Microverse Portfolio - feature: Desktop version
This PR was raised to add responsive web design using media queries.

# Description of the task to be completed
Used Github workflow
Configured the project to use linters
Added media queries to create responsive web design
Fixed inter errors
Added a descriptive PR

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,6 +24,15 @@
...
+            </svg>
+            </li>
+        </ul>
```
Comment (ahmetbozaci) [inline index.html:35]: - [ ] The original design has more left and right space on the navigation.  To stick to the design place change it.

### YOUR DESİGN:
image
___
### ORIGINAL DESİGN
image

File: index.html [hunk 2]
```diff
@@ -294,9 +321,10 @@ <h1 class="about-header">About me</h1>
         <li class="skill-lang-li">Gitlab</li>
         <li class="skill-lang-li">Terminal</li>
       </ul>
+      </div>
     </section>
 
-    <footer class="footer">
+    <section class="contact">
```
Comment (ahmetbozaci) [inline index.html:327]: - [ ] Text is not aligned like in the original design. Please fix it.
- [ ] In the desktop version `full name` become `first name and last name`. Kindly fix it.
- [ ] Also you should align the `get in touch` button like in the original
### YOUR
image
___
### ORIGINAL
image

File: index.html [hunk 3]
```diff
@@ -102,130 +108,151 @@ <h3 class="work-project-header">Multi-Post Stories</h3>
...
     </section>
 
     <!--About me-->
     <section class="about">
-      <h1 class="about-header">About me</h1>
-      <p class="about-description">
+      <div class="about-section">
+        <div class="about-sub-section">
+          <h1 class="about-header">About me</h1>
+          <p class="about-description">
```
Comment (ahmetbozaci) [inline index.html:240]: - [ ] Make sure the about-description has 4 lines like in the original. Kindly fix this to stick to the original design.

image
__
### ORIGINAL
image

File: index.html [hunk 4]
```diff
@@ -102,130 +108,151 @@ <h3 class="work-project-header">Multi-Post Stories</h3>
...
+      <div class="work-grid-container">
       <!--grid elements-->
       <!--grid elements 1-->
-      <div class="grid-project-container grid-1">
-        <h4 class="grid-project-header">Profesional Art Printing Data</h4>
-        <p class="grid-project-description">
+        <div class="grid-project-container grid-1">
+          <div class="grid-project1">
+            <h4 class="grid-project-header">Profesional Art Printing Data</h4>
+            <p class="grid-project-description">
```
Comment (ahmetbozaci) [inline index.html]: - [ ] In the first project is different from than others. Please change it. Also after change make sure buttons include `hover, active, press` styles.
### ORIGINAL DESKTOP
image
___

- [ ] In the mobile version `see project` buttons do not exist on your page. I checked the main branch buttons exist. Please fix it. This button should take you to the Live Demo of the project or to the GitHub repository.
### YOUR DESIGN MOBİLE VERSİON
image

File: index.html [hunk 5]
```diff
@@ -102,130 +108,151 @@ <h3 class="work-project-header">Multi-Post Stories</h3>
...
             </p>
+
+            <ul class="lang-list-ul">
+              <li class="lang-list">css</li>
+              <li class="lang-list">html</li>
+              <li class="lang-list">bootstrap</li>
+              <li class="lang-list">Ruby</li>
+            </ul>
+
+            <button type="button" class="project-btn">See Project</button>
```
Comment (ahmetbozaci) [inline index.html]: - [ ] This button should take you to the Live Demo of the project or to the GitHub repository. Please fix it.

File: index.html [hunk 6]
```diff
@@ -102,130 +108,151 @@ <h3 class="work-project-header">Multi-Post Stories</h3>
               ever since the 1500s, when an unknown printer took a standard
               dummy text.
             </p>
+
+            <ul class="lang-list-ul">
+              <li class="lang-list">css</li>
```
Comment (ahmetbozaci) [inline index.html:113]: _This inline comment is for lines 87-89.I cant choose the title of my recent work for inline comment. So please ignore the lines that ı choose_

- [ ] Please check `my recent works title and horizontal line. ` how is it in the original design. Kindly fix it.

image
ORIGINAL
image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️

### Hello @AkshithaReddy1899  ✋🏼 
#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- You know how to use media queries

___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [ ] Your commit messages should always start with capital letters and you should write an imperative tense. I see some of them are not like that. I write it as optional. But this is a [general requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=The%20commit%20message%20should%20be%20written%20in%20the%20imperative%20tense%20and%20capitalized) for every project. So you should pay attention to this for next projects.
Instead of writing `'Added style' or Created the file`' you should write `Add style'  or 'Create the file'`

**OPTIONAL**
- [ ] You can change them by using `git rebase`. If you want to know how to change ı leave a few resources. 
- [Video - How to change commit messages](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
- [Article - How to create good commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
___

### Cheers and Happy coding! 💻 👏

almost


___

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_


_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mentioning me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._
___
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: index.html (hunk 2 above)
Comment (AkshithaReddy1899) [reply index.html:327]: **The instructions clearly say to ignore the two input fields and use only one.**

File: index.html (hunk 4 above)
Comment (AkshithaReddy1899) [reply index.html]: **In the desktop version, the first tile is used to show the hover effects, it is not different from other tiles.**

File: index.html (hunk 2 above)
Comment (ahmetbozaci) [reply index.html:327]: @AkshithaReddy1899 You are right ı miss this part. You can ignore this.

**Review by ShoiraTa — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @AkshithaReddy1899 ,  

Your project is complete! Great job on working on changes requested by previous reviewer! Congratulations!  🎉💯🌟 



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**